docs/features/mergeStrategy.md

```diff
@@ -0,0 +1,27 @@
+# Merge Strategy
+
+It's possible to load multiple config files. CasC can load YAML files from a directory.
+And it's convenient to maintain if we split different parts of Jenkins into multiple files.
+
+## Support strategies
+
+* [IrreconcilableMergeStrategy]((../../plugin/src/main/java/io/jenkins/plugins/casc/yaml/IrreconcilableMergeStrategy.java)) (default)
```

Comment:
Shouldn't we just match the class name to the subtracting mergestrategy?

so the class should be called `ErrorOnConflictMergeStrategy`

Reply:
Sure.